tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -884,7 +875,7 @@ def supported_property_hash(tests, id, property_hash)
 # - Creates manifests
 # - Creates puppet resource title strings
 # - Cleans resource
-def test_harness_run(tests, id, skip_idempotence_check=false)
+def test_harness_run(tests, id, harness_dependencies=nil, skip_idempotence_check=false)
```

Comment:
to avoid the issues around renumbering arguments, you should use named params:

```ruby
def test_harness_run(tests, id, harness_dependencies: nil, skip_idempotence_check: false)
```